### PR TITLE
Connect Bookstore API

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Connected Bookstore API
- Refactor code in the application using middleware to make async requests to the API.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @gracekabaghe 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { React } from 'react';
-import './App.css';
+// import './App.css';
```
Comment (mikenath223) [inline src/App.js]: - [x] Kindly remove the commented code.

File: package.json [hunk 2]
```diff
@@ -7,14 +7,18 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-normalize": "^10.0.1",
+    "postcss-preset-env": "^7.0.2",
```
Comment (mikenath223) [inline package.json:12]: - [x] Please update the live link on the readme.

File: src/App.js (hunk 1 above)
Comment (gracekabaghe) [reply src/App.js]: thanks removed

File: package.json (hunk 2 above)
Comment (gracekabaghe) [reply package.json:12]: Live link updated in README.MD

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @gracekabaghe, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**